index.ts

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+import * as _ from 'lodash';
+import * as Serverless from "serverless";
+
+interface FetchResourcesParams {
+  data: any[];
+  nextToken: string;
+}
+
+class ServerlessCloudWatchLogsTagPlugin {
+
+  public serverless: Serverless;
+  public options: Serverless.Options;
+  public hooks: object;
+
+  public cloudFormationService: any;
+  public cloudWatchLogsService: any;
+  public lambdaService: any;
+  public awsService: any;
+
+  constructor(serverless, options) {
+
+    this.serverless = serverless;
+    this.options = options;
+    this.awsService = this.serverless.getProvider('aws');
+
+    this.cloudFormationService = new this.awsService.sdk.CloudFormation({region: this.options.region});
+    this.cloudWatchLogsService = new this.awsService.sdk.CloudWatchLogs({region: this.options.region});
+    this.lambdaService = new this.awsService.sdk.Lambda({region: this.options.region});
```

Comment:
use curly object spacing as specified in airbnb lint format

```suggestion
    this.cloudFormationService = new this.awsService.sdk.CloudFormation({ region: this.options.region });
    this.cloudWatchLogsService = new this.awsService.sdk.CloudWatchLogs({ region: this.options.region });
    this.lambdaService = new this.awsService.sdk.Lambda({ region: this.options.region });
```